### ISSUE TITLE
Please make the Preferences (front page) default theme to dark when we first unpack. This is hurting our eyes really. The overwhelming majority of people not prefer the color white

### ISSUE THREAD

**Issue by coffeechococake7** [TheFantasticWarrior/chrome-extension-imagus#45]

**Comment by ReEnMikki** [comment]
I second this. Dark theme gang ftw.

**Comment by DepressedOptimist** [comment]
Not sure if it is possible, but can you make it the same as what is selected on the desktop?

**Comment by kuzn123** [comment]
>> The overwhelming majority of people not prefer the color white

You are completely wrong. Most of the themes in programs and operating systems remain light by default, and dark themes are usually in the form of options.

**Comment by DepressedOptimist** [comment]
> > > The overwhelming majority of people not prefer the color white
> 
> You are completely wrong. Most of the themes in programs and operating systems remain light by default, and dark themes are usually in the form of options.

Actually, the majority of apps these days default to the OS' theme. I'm wondering if the browser's API can provide that info to extensions/add-ons.
